src/commands/MineSweeperCommand.ts

```diff
@@ -0,0 +1,48 @@
+import { Message, MessageEmbed } from "discord.js";
+import Command from "../abstracts/Command";
+import { EMBED_COLOURS } from "../config.json";
+import MineSweeperService from "../services/MineSweeperService";
+import StringUtils from "../utils/StringUtils";
+
+class MineSweeperCommand extends Command {
+	constructor() {
+		super(
+			"minesweeper",
+			"Generates a minesweeper game."
+		);
+	}
+
+	async run(message: Message, args: string[]) {
+		const embed = new MessageEmbed();
+		const mineSweeperService = MineSweeperService.getInstance();
+
+		if (!args[0]) args[0] = "easy";
+
+		interface DifficultyLevel {
+			[key: string]: {ratio: number}
+		}
+
+		const difficultyLevels: DifficultyLevel = {
+			"easy": {ratio: 7},
+			"medium": {ratio: 5},
+			"hard": {ratio: 3}
+		};
```

Comment:
Why is this an object of objects? The inner objects only ever have one key, you could just simplify this to:
```suggestion
		const difficultyLevels: DifficultyLevel = {
			easy: 7,
			medium: 5,
			hard: 3
		};
```